README.md

```diff
@@ -8,50 +8,67 @@ You must install the Emacs plugin (this) and a language server
 
 ### lsp-scala
 
-Install the `lsp-mode` and `sbt-mode` dependencies using your preferred package
-manager. Then clone this repo and load it:
+Clone this repo, and fetch `lsp-mode`, `lsp-ui`, and `sbt-mode` from
+MELPA with the package manager of your choice.  (We are working on
+adding this package to MELPA to make `lsp-scala` load like the rest.)
+Here is an example using `use-package`:
 
 ```emacs-lisp
-(add-to-list 'load-path "<path to lsp-scala>")
-(require 'lsp-scala)
+(use-package lsp-mode
+  :ensure t)
+
+(use-package lsp-scala
+  :load-path "~/path/to/lsp-scala"
+  :config
+  (setq lsp-scala-server-command "~/bin/metals-emacs")
+  (require 'lsp)
```

Comment:
I don't think you need to explicitly `require`. `lsp` is an autoload, so it'll be required when needed.